### ISSUE TITLE
Clan balance gains extra money on creation

### ISSUE THREAD

**Issue by zellfrey** [The-Fireplace-Minecraft-Mods/Clans#42]
http://prntscr.com/nbkje1

I have an initial balance. When i form a clan it deducts the set amount of credits (i looked through the source and im confident that works accordingly) 

` I:formClanBankAmount=5000`

    # Cost of forming a clan. This requires a compatible economy to be installed.
    # Min: 0
    # Max: 2147483647
    I:formClanCost=100`

However when the clan is formed i seem to get an extra $175?

This is my config setting for GrandEconomy
`# Amount of credits given to new players joining the server [range: 0 ~ 10000, default: 100]
    I:startBalance=200`

So i did a bit of digging. It seems on the New  clan creation/construction it creates a new Grandeconomy account?

`Clans.getPaymentHandler().ensureAccountExists(clanId);`

```
public void ensureAccountExists(UUID account) {
		Account.get(account);
	}
```
To which it then adds the config amount to this new GrandEconomy account;

` Clans.getPaymentHandler().addAmount(Clans.cfg.formClanBankAmount, clanId);`

So in theory its an extra $200. However i instead end up with an extra $175 so i don't know whats happening to deduct $25.

Anyway to sum up, i get an extra $175 on clan creation on top of formClanBankAmoun

Clans-1.12.2-1.2.3
GrandEconomy-1.12.2-1.0.1

Forge version :1.12.2-14.23.5.2781

Edit: Also side note, with /clan details giving details of clan, what was the decision of having the clan balance not being displayed on the details?

**Comment by The-Fireplace** [comment]
My thought process was that there is a separate command that shows a clan's balance, but only for clans you're in. Just like you can't look at how much is in other players' wallets, you can't look in clans' wallets unless you're in the clan.
